site/en/blog/crx-scripting-api/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Introducing the chrome.scripting API"
+# description: >
+#   Why and how we implemented color vision deficiency simulation in DevTools and the Blink Renderer.
+layout: "layouts/blog-post.njk"
+authors:
+  - dotproto
+date: 2021-05-03
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/HD1U7BxDVRDcTxNaQJ4C.jpg'
+alt: ''
+tags:
+  - extensions
+---
+
+Manifest V3 introduces a number of changes to Chrome's extension platform, a handful of which are
+notable or significant changes. In this post, we'll be exploring the motivations and changes
```

Comment:
nit: this implies that many / most of the changes in MV3 are not notable : )

Reply:
Hm. I think it's easiest to just drop that clause.